### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by troleksii**
- [DEMO LINK](https://troleksii.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://troleksii.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -2,13 +2,48 @@
...
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="/" class="moyo__logo">
+        <img src="./images/logo.png" alt="moyo_logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="add__marg">
```
Comment (Viktor-Kost) [inline src/index.html:20]: change a class name
```suggestion
          <li class="nav__item">
```

File: src/index.html [hunk 2]
```diff
@@ -2,13 +2,48 @@
...
+          </li>
+          <li class="add__marg">
+            <a class="nav__link" href="#samsung">samsung</a>
+          </li>
+          <li class="add__marg">
+            <a class="nav__link" href="#smartphones">smartphones</a>
+          </li>
+          <li class="add__marg">
+            <a class="nav__link" data-qa="hover"
+             href="#laptops_&_computers">laptops & computers</a>
```
Comment (Viktor-Kost) [inline src/index.html:31]: code styling
```suggestion
            <a 
              class="nav__link" 
              data-qa="hover"
              href="#laptops_&_computers"
            >
              laptops & computers
            </a>
```

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,93 @@
 body {
   margin: 0;
+  background-color: grey;
+  font-family: Roboto,sans-serif;
+
+  --header-height: 60px;
```
Comment (Viktor-Kost) [inline src/style.css:6]: no needs to hardcode height
```suggestion
```

File: src/style.css [hunk 4]
```diff
@@ -1,3 +1,93 @@
...
+  --active-color: #00ACDC;
```
Comment (Viktor-Kost) [inline src/style.css:7]: it's netter to move variables to the `root`

File: src/style.css [hunk 5]
```diff
@@ -1,3 +1,93 @@
...
+}
+
+.header {
+  display: flex;
+  position: relative;
```
Comment (Viktor-Kost) [inline src/style.css:12]: shouldn't be relative
```suggestion
```

File: src/style.css [hunk 6]
```diff
@@ -1,3 +1,93 @@
...
+  justify-content: space-between;
+  align-items: center;
+  flex-shrink: 0;
+  background: #FFF;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+
```
Comment (Viktor-Kost) [inline src/style.css:18]: add `padding: 0 50px;` to avoid using positioning

File: src/style.css [hunk 7]
```diff
@@ -1,3 +1,93 @@
...
+}
+
+.moyo__logo {
+  display: flex;
+  position: relative;
+  left: 50px;
```
Comment (Viktor-Kost) [inline src/style.css:24]:
```suggestion
  display: block;
```

File: src/style.css [hunk 8]
```diff
@@ -1,3 +1,93 @@
...
+}
+
+.nav {
+  display: flex;
+  position: relative;
+  right: 50px;
+
+}
```
Comment (Viktor-Kost) [inline src/style.css:32]: redundant
```suggestion
```

File: src/style.css [hunk 9]
```diff
@@ -1,3 +1,93 @@
...
+
+.nav__list {
+  display: flex;
+  flex-shrink: 0;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  position: relative;
```
Comment (Viktor-Kost) [inline src/style.css:40]:
```suggestion
```

File: src/style.css [hunk 10]
```diff
@@ -1,3 +1,93 @@
...
+}
+
+
+
```
Comment (Viktor-Kost) [inline src/style.css:44]:
```suggestion
```

File: src/style.css [hunk 11]
```diff
@@ -1,3 +1,93 @@
...
+.nav__link {
+  color: #000;
+  text-transform: uppercase;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: var(--header-height);
```
Comment (Viktor-Kost) [inline src/style.css:50]: set a correct line height from figma and to get height you need you can just use padding

File: src/style.css [hunk 12]
```diff
@@ -1,3 +1,93 @@
...
+  text-decoration: none;
+  display: block;
+  position: relative;
```
Comment (Viktor-Kost) [inline src/style.css:53]: move to `is-active`

File: src/style.css [hunk 13]
```diff
@@ -1,3 +1,93 @@
...
+}
+
+.add__marg {
+  margin-right: 20px;
+}
+
+.add__marg:last-child {
+  margin-right: 0;
+}
```
Comment (Viktor-Kost) [inline src/style.css:62]:
```suggestion
.nav__item:not(:last-child) {
  margin-right: 20px;
}
```

File: src/style.css [hunk 14]
```diff
@@ -1,3 +1,93 @@
...
+
+.is-active {
+  color: #00ACDC;
```
Comment (Viktor-Kost) [inline src/style.css:65]: reuse a color variable

File: src/style.css [hunk 15]
```diff
@@ -1,3 +1,93 @@
...
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  bottom: 0;
+  height: 4px;
+  width: 100%;
+  flex-shrink: 0;
+  background: #00ACDC;
```
Comment (Viktor-Kost) [inline src/style.css:79]: color variable

File: src/style.css [hunk 16]
```diff
@@ -1,3 +1,93 @@
...
+@media (max-width: 1024px) {
+  .header {
+    width: 1024px;
+  }
+}
+
+@media (min-width: 1025px) {
+  .header {
+    width: 1200px;
+  }
```
Comment (Viktor-Kost) [inline src/style.css:92]: no needs
```suggestion
```

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 12 above)
Comment (troleksii) [reply src/style.css:53]: @Viktor-Kost  when I moved that part to ".active" selector - nav links moved higher and lost style. should I move it?
Знімок екрана 2024-01-22 о 22 28 15

File: src/index.html (hunk 1 above)
Comment (Viktor-Kost) [reply src/index.html]: not only to the first  `li` but for other ones as well

File: src/style.css (hunk 12 above)
Comment (Viktor-Kost) [reply src/style.css:53]: You should also apply fixes from other comments to have a correct solution

`position: relative;` should be added only to `.is-active` selector.
So your pseudo-element `:after` will have `position: absolute;`

If you need any help with this issue feel free to ask at fe_chat

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]
Nice!
I see no changes. check either you've push your commit or not
If you need any help with this issue feel free to ask at fe_chat

**Review by etojeDenys — APPROVED** [review]
good job

**Pull request closed without merging**